lib/bolt/validator.rb

```diff
@@ -0,0 +1,224 @@
+# frozen_string_literal: true
```

Comment:
Moved this out of the `Bolt::Config` namespace since it's used by more than just that class.